### PR TITLE
Add Hello World in Java

### PR DESCRIPTION
**Pull request by Janaki2399**

### CODE REVIEW

File: code/Java/Hw.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import java.io.*;
+import java.util.*;
+
+
+class HelloWorld
+{
```
Comment (adiaholic) [inline code/Java/Hw.java:4]: Looks like you are going about this workflow perfectly. However I strongly recommend using [Google Java Style Guide](https://google.github.io/styleguide/javaguide.html) for programming in Java. In bigger organisations, code maintenance is as important as adding newer modules.
Congrats on your very first PR 😃!

**Review by adiaholic — CHANGES_REQUESTED** [review]

**Comment by adiaholic** [comment]
Squash please

**Comment by Janaki2399** [comment]
> Squash please

I tried squashing. How do I check if its squashed?

**Comment by Janaki2399** [comment]
I sqashed it

**Comment by Janaki2399** [comment]
I squashed the commits

**Review by adiaholic — APPROVED** [review]

**Pull request merged**